default/js/theme/templates.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @author ps / @___paul
+ */
+
+'use strict';
+
+const nunjucks = require("nunjucks/browser/nunjucks-slim");
+const settings = window.LB.settings;
+
+const defaultTemplates = {
+  post: require("../../templates/template-post.html"),
+  timeline: require("../../templates/template-timeline.html"),
+  itemImage: require("../../templates/template-item-image.html"),
+  itemEmbed: require("../../templates/template-item-embed.html")
+};
+
+function getCustomTemplates() {
+  let customTemplates = settings.customTemplates
```

Comment:
@m0g let's remind this when we'll add template extension on sample extended theme.